### PR TITLE
#4963 [Manager ] Delete topic button should be disabled when the topi…

### PR DESCRIPTION
**Pull request by eechih**
…c is being deleted

### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #4963 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [ ] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/Topic/TopicDeleteDialog.js [hunk 1]
```diff
@@ -33,14 +33,35 @@ function TopicDeleteDialog({
...
+        })
+        .catch(() => {
+          setLoading(false);
+          onClose();
+        });
+    } else {
+      setLoading(false);
+      onClose();
+    }
+  };
```
Comment (saivirtue) [inline ohara-manager/client/src/components/Topic/TopicDeleteDialog.js:55]: 看起來是想要透過Epic成功後，觸發resolve()動作來把按鈕重新enable?
這裡有個想法，如果是透過在ui.topics裡將`loading`屬性改變來判斷呢?
```
case actions.deleteTopic.TRIGGER:
      return {
        ...state,
        loading: true,
        error: null,
      };
case actions.deleteTopic.SUCCESS:
      return {
        ...state,
        loading: false,
        lastUpdated: new Date(),
      };
```

Comment (eechih) [reply ohara-manager/client/src/components/Topic/TopicDeleteDialog.js:55]: 這個建議很好～不過刪除 Topic 的過程包含了兩個動作，先 stop 再 delete，loading 應該從 stop trigger 開始持續到 delete success 結束，所以 stop success 與 delete trigger 會中斷 loading。因此我不採取 redux 的作法。

**Comment by eechih** [comment]
retry manager-e2e

**Comment by konekoya** [comment]
retry manager-e2e

**Pull request merged**